src/PlaywrightEnvironment.ts

```diff
@@ -136,6 +136,37 @@ export const getPlaywrightEnv = (basicEnv = 'node'): unknown => {
         this.global.page.on('pageerror', handleError)
       }
       this.global.jestPlaywright = {
+        resetPage: async (): Promise<void> => {
+          const { context, page } = this.global
+          if (page) {
+            page.removeListener('pageerror', handleError)
+            await page.close()
+          }
+
+          this.global.page = await context.newPage()
+          if (exitOnPageError) {
+            this.global.page.addListener('pageerror', handleError)
+          }
+        },
+        resetBrowser: async (): Promise<void> => {
+          const { browser } = this.global
+
+          if (browser) {
+            await browser.close()
+          }
+
+          this.global.browser = await getBrowserPerProcess(
```

Comment:
I personally think its too heavy to restart a whole browser there. Since the browser/context/page was restructured in Playwright compared to Puppeteer, I would rename it to resetContext and only create a new context and close the old one. Or also add it additionally. Not sure.

Reply:
@mxschmitt I can can add `resetContext`, we can also keep `resetBrowser`. But we can add some info in documentation that it is heavy method

Reply:
@mxschmitt just added `resetContext`